cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/compute/GcpInstanceResourceBuilder.java

```diff
@@ -482,13 +482,19 @@ private List<CloudResource> filterResourcesByType(List<CloudResource> resources,
                 .collect(Collectors.toList());
     }
 
-    private List<CloudResource> filterGroupFromName(List<CloudResource> resources, String filterString) {
+    private List<CloudResource> filterGroupFromName2(List<CloudResource> resources, String filterString) {
```

Comment:
it seems to be unused :slightly_smiling_face:

Reply:
alrighty